arch/arm/core/aarch64/isr_wrapper.S

```diff
@@ -111,13 +111,6 @@ SECTION_FUNC(TEXT, _isr_wrapper)
 	/* Switch thread */
```

Comment:
Now that the actual switch takes place at  `z_arm64_exit_exc`  should we elaborate it in comment.  May be somewhere in comment we should mention the flow.

Reply:
ok. I'll also noticed a bunch of other comments that need to be updated. I'll add a comments fixing PR on top of the next respin.